Sources/PointFree/Pricing.swift

```diff
@@ -10,6 +10,16 @@ import Prelude
 import Styleguide
 import Tuple
 
+public let teamMinDiscountAt = 5
+public let teamMidDiscountAt = 10
+public let teamMaxDiscountAt = 20
+
+// FIXME: Fetch from Stripe?
+public let teamBaseSeatPriceMonthly = 13
+public let teamMinDiscountRate = 0.95
+public let teamMidDiscountRate = 0.9
+public let teamMaxDiscountRate = 0.85
```

Comment:
This stuff is a lil messy, but good enough for now!

Reply:
would it be simpler to have 3 separate plans for teams? `team-yearly-2-to-4`, `team-yearly-2-to-9`, `team-yearly-10-and-up`?